iohk/NixOps.hs

```diff
@@ -315,15 +336,18 @@ instance ToJSON NixopsConfig where
 
 deploymentFiles :: Environment -> Target -> [Deployment] -> [Text]
 deploymentFiles cEnvironment cTarget cElements =
+  "deployments/firewalls.nix":
```

Comment:
let's keep Amazon terminology: `securitygroups.nix`.

Also, this file should only be used for `Nodes` and `Explorer`